lab2.ipynb

```diff
@@ -637,42 +516,17 @@
-      "text/plain": [
-       "<Figure size 432x288 with 1 Axes>"
-      ]
-     },
-     "metadata": {
-      "needs_background": "light"
-     },
-     "output_type": "display_data"
-    }
-   ],
+   "outputs": [],
    "source": [
```

Comment:
Couldn't find the top place to make this comment: but for this problem, 2A how the distribution changes, since we are asked about the average, your x-axis should be scaled by a factor of 1/num_convolutions